api/Worker.json

```diff
@@ -6,7 +6,8 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/workers.html#dedicated-workers-and-the-worker-interface",
         "support": {
           "chrome": {
-            "version_added": "2"
+            "version_added": "2",
+            "notes": "Not available in Shared Workers. See <a href='https://crbug.com/40695450'>bug 40695450</a>."
```

Comment:
Does the spec indicate that `Worker` should be available in Shared Workers? I can see that there is a separate `SharedWorker` interface.

Can you explain what it is that say Firefox supports here but Chrome doesn't?